### ISSUE TITLE
8.2.13. New - Screen Name/Title: Create a service - Radio Button Group Lacks Clear Required Indicator

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-planning#2461]
Component: Create template - Will you send the message by email or text?
Observation:
Radio Button Group Lacks Clear Required Indicator
While visually it may be obvious that a selection is required, there is no programmatic or visible indication that the radio button group is mandatory.
Screen reader users do not receive feedback that the field is required.
Cognitive users may overlook the requirement, especially if there is no validation message.

(Attachment: 8.2.13. Radio Button Group Lacks Clear Required Indicator.png)

Recommendation: 
Add a visible indicator like an asterisk (*) with explanatory text (* required)
Programmatically mark inputs as required with either:
<input ... required>
or
<input aria-required="true" ...>

**Comment by whabanks** [comment]
Related to #2286. If we're adding required marks to text fields we should probably include radio button groups as well.
